hydroflow/tests/surface_codegen.rs

```diff
@@ -415,6 +416,75 @@ pub fn test_groupby() {
     println!();
 }
 
+#[test]
+pub fn test_demux_1() {
+    enum Shape {
+        Circle(f64),
+        Rectangle { width: f64, height: f64 },
+        Square(f64),
+    }
+
+    let mut df = hydroflow_syntax! {
+        demux = recv_iter([
```

Comment:
Confusing to use variable names that are the same as keywords, even if it's legal.